C3 - Nuxt failures

### Which Cloudflare product(s) does this pertain to?

C3

### What version(s) of the tool(s) are you using?

latest

### What version of Node are you using?

18

### What operating system and version are you using?

Mac

### Describe the Bug

The Nuxt application created via C3 appears to be broken. This can be seen in failing C3 e2e tests but also by locally running C3 and creating a Nuxt app.

### Please provide a link to a minimal reproduction

_No response_

### Please provide any relevant error logs

```
pnpm run preview  

> nuxt-app@ preview /Users/pbacondarwin/dev/temp/muddy-dust-6087
> pnpm run build && wrangler pages dev


> nuxt-app@ build /Users/pbacondarwin/dev/temp/muddy-dust-6087
> nuxt build

[10:22:34 PM] Nuxt 3.11.2 with Nitro 2.9.6
[10:22:35 PM] ℹ Building client...
[10:22:35 PM] ℹ vite v5.3.1 building for production...
[10:22:36 PM] ℹ ✓ 110 modules transformed.
[10:22:36 PM] ℹ .nuxt/dist/client/manifest.json                   1.94 kB │ gzip:  0.42 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/error-500.BY691cIp.css    1.88 kB │ gzip:  0.72 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/error-404.Cs7Z0ZwI.css    3.56 kB │ gzip:  1.10 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/entry.DYp2-o9r.css       12.40 kB │ gzip:  2.56 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/pUFXoEQj.js               2.74 kB │ gzip:  1.23 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/CZWyxFol.js               7.38 kB │ gzip:  3.08 kB
[10:22:36 PM] ℹ .nuxt/dist/client/_nuxt/CZGI4OxP.js             210.36 kB │ gzip: 60.31 kB
[10:22:36 PM] ℹ ✓ built in 778ms
[10:22:36 PM] ✔ Client built in 786ms
[10:22:36 PM] ℹ Building server...
[10:22:36 PM] ℹ vite v5.3.1 building SSR bundle for production...
[10:22:37 PM] ℹ ✓ 123 modules transformed.
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/entry-styles.DCzG2Gh4.mjs               0.08 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-404-styles.CGceJTXr.mjs           0.08 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-500-styles.CxCcaIEE.mjs           0.08 kB
[10:22:38 PM] ℹ .nuxt/dist/server/styles.mjs                                    1.57 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-500-styles-1.mjs-seeEwM8D.js      2.10 kB │ map:     0.12 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-404-styles-1.mjs-DKAPqvyK.js      3.80 kB │ map:     0.12 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-500-BsL1lVhm.js                   3.98 kB │ map:     4.64 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/multipart-parser-ngo8Vv15.js            6.87 kB │ map:    10.87 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/error-404-B4y3Ixwj.js                  11.73 kB │ map:    23.25 kB
[10:22:38 PM] ℹ .nuxt/dist/server/_nuxt/entry-styles-1.mjs-BoX6ESXr.js         12.65 kB │ map:     0.11 kB
[10:22:38 PM] ℹ .nuxt/dist/server/server.mjs                                1,534.98 kB │ map: 2,520.16 kB
[10:22:38 PM] ℹ ✓ built in 1.71s
[10:22:38 PM] ✔ Server built in 1718ms
[nitro 10:22:38 PM] ✔ Generated public dist
[nitro 10:22:38 PM] ℹ Building Nuxt Nitro server (preset: cloudflare-pages)
[nitro 10:22:43 PM] ✔ Nuxt Nitro server built
  ├─ dist/_worker.js/chunks/_/error-500.mjs (4.82 kB) (2.05 kB gzip)
  ├─ dist/_worker.js/chunks/_/error-500.mjs.map (358 B) (227 B gzip)
  ├─ dist/_worker.js/chunks/build/client.manifest.mjs (2.15 kB) (519 B gzip)
  ├─ dist/_worker.js/chunks/build/client.manifest.mjs.map (931 B) (402 B gzip)
  ├─ dist/_worker.js/chunks/build/entry-styles.DCzG2Gh4.mjs (12.5 kB) (2.62 kB gzip)
  ├─ dist/_worker.js/chunks/build/entry-styles.DCzG2Gh4.mjs.map (283 B) (183 B gzip)
  ├─ dist/_worker.js/chunks/build/error-404-B4y3Ixwj.mjs (6.4 kB) (2.58 kB gzip)
  ├─ dist/_worker.js/chunks/build/error-404-B4y3Ixwj.mjs.map (5.72 kB) (2.54 kB gzip)
  ├─ dist/_worker.js/chunks/build/error-404-styles.CGceJTXr.mjs (3.68 kB) (1.18 kB gzip)
  ├─ dist/_worker.js/chunks/build/error-404-styles.CGceJTXr.mjs.map (298 B) (189 B gzip)
  ├─ dist/_worker.js/chunks/build/error-500-BsL1lVhm.mjs (3.03 kB) (1.36 kB gzip)
  ├─ dist/_worker.js/chunks/build/error-500-BsL1lVhm.mjs.map (1.27 kB) (694 B gzip)
  ├─ dist/_worker.js/chunks/build/error-500-styles.CxCcaIEE.mjs (1.98 kB) (797 B gzip)
  ├─ dist/_worker.js/chunks/build/error-500-styles.CxCcaIEE.mjs.map (298 B) (187 B gzip)
  ├─ dist/_worker.js/chunks/build/multipart-parser-ngo8Vv15.mjs (4.67 kB) (2 kB gzip)
  ├─ dist/_worker.js/chunks/build/multipart-parser-ngo8Vv15.mjs.map (6.48 kB) (2.66 kB gzip)
  ├─ dist/_worker.js/chunks/build/server.mjs (877 kB) (254 kB gzip)
  ├─ dist/_worker.js/chunks/build/server.mjs.map (107 kB) (34 kB gzip)
  ├─ dist/_worker.js/chunks/build/styles.mjs (1.52 kB) (447 B gzip)
  ├─ dist/_worker.js/chunks/build/styles.mjs.map (513 B) (267 B gzip)
  ├─ dist/_worker.js/chunks/routes/renderer.mjs (99.8 kB) (34 kB gzip)
  ├─ dist/_worker.js/chunks/routes/renderer.mjs.map (22.6 kB) (7.77 kB gzip)
  ├─ dist/_worker.js/chunks/runtime.mjs (112 kB) (34.1 kB gzip)
  ├─ dist/_worker.js/chunks/runtime.mjs.map (22.3 kB) (7.34 kB gzip)
  ├─ dist/_worker.js/chunks/virtual/_virtual_spa-template.mjs (85 B) (99 B gzip)
  ├─ dist/_worker.js/chunks/virtual/_virtual_spa-template.mjs.map (108 B) (108 B gzip)
  └─ dist/_worker.js/index.js (139 B) (144 B gzip)
Σ Total size: 1.3 MB (392 kB gzip)
[nitro 10:22:44 PM] ✔ You can preview this build using npx wrangler pages dev dist/
[nitro 10:22:44 PM] ✔ You can deploy this build using npx wrangler pages deploy dist/

 ⛅️ wrangler 3.62.0
-------------------

Attaching additional modules:
┌────────────────────────────────────────────┬──────┬────────────┐
│ Name                                       │ Type │ Size       │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/_/error-500.mjs                     │ esm  │ 4.71 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/client.manifest.mjs           │ esm  │ 2.10 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/entry-styles.DCzG2Gh4.mjs     │ esm  │ 12.24 KiB  │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/error-404-B4y3Ixwj.mjs        │ esm  │ 6.25 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/error-404-styles.CGceJTXr.mjs │ esm  │ 3.59 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/error-500-BsL1lVhm.mjs        │ esm  │ 2.96 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/error-500-styles.CxCcaIEE.mjs │ esm  │ 1.93 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/multipart-parser-ngo8Vv15.mjs │ esm  │ 4.56 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/server.mjs                    │ esm  │ 856.14 KiB │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/build/styles.mjs                    │ esm  │ 1.48 KiB   │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/routes/renderer.mjs                 │ esm  │ 97.43 KiB  │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/runtime.mjs                         │ esm  │ 109.20 KiB │
├────────────────────────────────────────────┼──────┼────────────┤
│ chunks/virtual/_virtual_spa-template.mjs   │ esm  │ 0.08 KiB   │
└────────────────────────────────────────────┴──────┴────────────┘
✨ Compiled Worker successfully
[wrangler:inf] Ready on http://localhost:8788
▲ [WARNING] The package "node:process" wasn't found on the file system but is built into node.

  Your Worker may throw errors at runtime unless you enable the
  "nodejs_compat" compatibility flag. Refer to
  https://developers.cloudflare.com/workers/runtime-apis/nodejs/ for more
  details. Imported from:
   - .wrangler/tmp/pages-V3rMpr/chunks/build/server.mjs


▲ [WARNING] The package "node:stream/web" wasn't found on the file system but is built into node.

  Your Worker may throw errors at runtime unless you enable the
  "nodejs_compat" compatibility flag. Refer to
  https://developers.cloudflare.com/workers/runtime-apis/nodejs/ for more
  details. Imported from:
   - .wrangler/tmp/pages-V3rMpr/chunks/build/server.mjs


⎔ Starting local server...
✨ Parsed 0 valid redirect rules.
✨ Parsed 3 valid header rules.
✘ [ERROR] [node-fetch-native] Node.js compatible build of `node-fetch-native` is being used in a non-Node.js environment. Please make sure you are using proper export conditions or report this issue to https://github.com/unjs/node-fetch-native. You can set `process.env.DISABLE_NODE_FETCH_NATIVE_WARN` to disable this warning.


✘ [ERROR] [nuxt] [request error] [unhandled] [500] __VUE_OPTIONS_API__ is not defined

    at normalizePropsOptions
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:41836)
    at createComponentInstance$1
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:66987)
    at renderComponentVNode
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:77437)
    at renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:81356)
    at Object.renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:94600)
    at Object.renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:3826)
    at async Tn2
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:96217)
    at async Object.handler
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:98552)
    at async Object.handler
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/runtime.mjs:1:72221)
    at null.<anonymous> (async
  file:.//Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/dev-M4BDRW/reclwnnr3qa.js:24640:7)


✘ [ERROR] [nuxt] [request error] [unhandled] [500] __VUE_OPTIONS_API__ is not defined

    at normalizePropsOptions
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:41836)
    at createComponentInstance$1
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:66987)
    at renderComponentVNode
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:77437)
    at renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:81356)
    at Object.renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:94600)
    at Object.renderToString
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:3826)
    at async Tn2
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:96217)
    at async Object.handler
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/routes/renderer.mjs:1:98552)
    at async Object.handler
  (file:./Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/pages-V3rMpr/chunks/runtime.mjs:1:72221)
    at null.<anonymous> (async
  file:.//Users/pbacondarwin/dev/temp/muddy-dust-6087/.wrangler/tmp/dev-M4BDRW/reclwnnr3qa.js:24640:7)


[wrangler:inf] GET / 500 Internal Server Error (43ms)
```

--- a/packages/create-cloudflare/e2e-tests/frameworks.test.ts
+++ b/packages/create-cloudflare/e2e-tests/frameworks.test.ts
@@ -266,7 +266,6 @@ const frameworkTests: Record<string, FrameworkTestConfig> = {
 	nuxt: {
 		testCommitMessage: true,
 		timeout: LONG_TIMEOUT,
-		quarantine: true,
 		unsupportedOSs: ["win32"],
 		verifyDeploy: {
 			route: "/",

